africanus/gridding/nifty/dask.py

```diff
@@ -347,7 +347,7 @@ def grid(vis, uvw, flags, weights, frequencies, grid_config,
 
 def _nifty_dirty(grid, grid_config):
     """ Wrapper function for creating a dirty image """
-    grids = [grid_config.grid2dirty_c(grid[:, :, c])
+    grids = [grid_config.grid2dirty(grid[:, :, c].real)
```

Comment:
I think this is a misunderstanding (but I'm not absolutely sure).
If you are using the `_c` functions everywhere else, you should be calling `grid2dirty_c` here and take the real part of the result.

Reply:
Yes I think you're right, I've fixed this.